parquet-avro/src/main/java/org/apache/parquet/avro/AvroConverters.java

```diff
@@ -341,27 +340,4 @@ public String convert(Binary binary) {
       return stringifier.stringify(binary);
     }
   }
-
-  static final class FieldVariantConverter<T> extends VariantColumnConverter {
```

Comment:
I moved this to its own class, `AvroVariantConverter`. I also merged it with the `VariantColumnConverter`. We can still move the logic around when it is reused but I thought it was cleaner to start with one and then determine how to share code when we have more uses.